packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -29,26 +34,51 @@ export const styles = (theme) => ({
     ...theme.typography.body2,
     minHeight: 'auto',
   },
+  indicatorWrapper: {
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  indicator: {
+    marginLeft: theme.spacing(2)
+  }
 });
 
 const MenuItem = React.forwardRef(function MenuItem(props, ref) {
   const {
+    children: childrenProp,
     classes,
     className,
     component = 'li',
     disableGutters = false,
+    nestedItems,
+    NestedMenuIndicator = KeyboardArrowRight,
+    NestedMenuProps = {},
+    openNestedMenu = false,
+    onKeyDown,
     role = 'menuitem',
     selected,
+    handleArrowRightKeydown,
+    setParentLastEnteredItemIndex,
     tabIndex: tabIndexProp,
     ...other
   } = props;
 
+  const listItemRef = React.useRef(null);
+  const handleOwnRef = React.useCallback((instance) => {
+    // #StrictMode ready
+    listItemRef.current = ReactDOM.findDOMNode(instance);
+  }, []);
+  const handleRef = useForkRef(handleOwnRef, ref);
+
   let tabIndex;
   if (!props.disabled) {
     tabIndex = tabIndexProp !== undefined ? tabIndexProp : -1;
   }
-  return (
+
+  const ListItemAndNestedMenu = [
     <ListItem
+      key="ListItem"
```

Comment:
We should dynamically set this somehow, I cant find any other instances of a fixed `key` value